go/vt/vttablet/tabletmanager/vreplication/table_plan_builder.go

```diff
@@ -28,6 +28,21 @@ import (
 	querypb "vitess.io/vitess/go/vt/proto/query"
 )
 
+const (
```

Comment:
These constanst should not be moved. They are defined right after their type, which makes it more readable.